src/stubs.ts

```diff
@@ -186,6 +199,17 @@ export function stubComponents(
       ]
     }
 
+    // stub teleport by default via config.global.stubs
+    if ((type as typeof Teleport).__isTeleport && 'teleport' in stubs && stubs['teleport']) {
```

Comment:
Can't we just do `type === Teleport`? Same as we do with `Transition`.

Reply:
At first that didn't work, because Teleport was not included in the `const type = nodeType as VNodeTypes` declaration.

It's kind of odd, because `VNodeTypes` literally has `| typeof Teleport` in the type, but now that I explicitly added `const type = nodeType as (VNodeTypes | typeof Teleport)` it does seem to work, let me try that again.